tests/topotests/ospf_te_topo1/test_ospf_te_topo1.py

```diff
@@ -214,7 +214,7 @@ def test_step4():
 
     tgen.net["r1"].cmd('vtysh -c "conf t" -c "router ospf" -c "segment-routing on"')
     tgen.net["r1"].cmd(
-        'vtysh -c "conf t" -c "router ospf" -c "segment-routing global-block 20000 23999"'
+        'vtysh -c "conf t" -c "router ospf" -c "segment-routing global-block 20000 23999 local-block 5000 6999"'
```

Comment:
The previous code was configuring r2 with local block and not r1. Please check.

Reply:
:+1:

Reply:
I confirm. This line 217 should not be modified while lines 226 - 228 should be kept and modified.